api/src/test/java/uk/gov/dhsc/htbhf/claimant/ChildrenDateOfBirthMatchIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.ResponseEntity;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimDTO;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimResultDTO;
+import uk.gov.dhsc.htbhf.claimant.testsupport.RepositoryMediator;
+import uk.gov.dhsc.htbhf.claimant.testsupport.WiremockManager;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.dhsc.htbhf.TestConstants.*;
+import static uk.gov.dhsc.htbhf.claimant.ClaimantServiceAssertionUtils.buildClaimRequestEntity;
+import static uk.gov.dhsc.htbhf.claimant.model.ClaimStatus.NEW;
+import static uk.gov.dhsc.htbhf.claimant.model.ClaimStatus.REJECTED;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimDTOTestDataFactory.aValidClaimDTOWithExpectedDeliveryDateAndChildrenDob;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.TestConstants.EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@AutoConfigureEmbeddedDatabase
+public class ChildrenDateOfBirthMatchIntegrationTest {
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Autowired
+    private WiremockManager wiremockManager;
+
+    @Autowired
+    private RepositoryMediator repositoryMediator;
+
+    @BeforeEach
+    void init() {
+        wiremockManager.startWireMock();
+    }
+
+    @AfterEach
+    void cleanup() {
+        wiremockManager.stopWireMock();
+        repositoryMediator.deleteAllEntities();
+    }
+
+    @ParameterizedTest
+    @MethodSource("providedArgumentsForCreateNewClaim")
+    void shouldCreateNewClaim(LocalDate expectedDeliveryDate,
```

Comment:
`shouldCreateNewClaimWhenPregnantOrChildDatesOfBirthMatch`

Reply:
I would argue whether we even need this as an extra integration test as its just the happy, surely we just need to add `assertThat(response.getBody().getVerificationResult().getIsPregnantOrAtLeast1ChildMatched()).isTrue();` to the existing happy path integration test we just have the one negative test.

Reply:
I guess it depends if we want an integration test for each decision path. Personally I'm in favour of more integration tests.

Reply:
One thing to bear in mind - the tests take 8 minutes or more on Adeel's machine.

Reply:
There are certainly pros and cons for each side of this argument, I don't feel too strongly either way so happy if it stays.